CONTRIBUTING.md

```diff
@@ -137,3 +137,23 @@ func main() {
     fmt.Println(sortedArray)
 }
 ```
+
+#### JavaScript
+
+```JavaScript
+function quickSort (arr) {
+    /*
```

Comment:
Don't get the point of TAB indentation of size 2 **THROUGHOUT**.
All the above samples have space indentation with zero throughout TAB indentation